### PR TITLE
Rewrite: Accessibility Guide

### PR DESCRIPTION
**Pull request by estelle**
Everything on the page was outdated, so I rewrote it. It doesn't have to be perfect, but we definitely needed to get rid of what was there.

This PR…
-->
- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/accessibility/information_for_web_authors/index.md [hunk 1]
```diff
@@ -7,63 +7,48 @@ tags:
...
 ## Guidelines and Regulations
 
-1. [Section 508 Guidelines for web authors](https://www.section508.gov/index.cfm?FuseAction=Content&#x26;ID=12#Web)
+1. [<abbr>ARIA</abbr> Authoring Practices Guide (<abbr>APG</abbr>) ](https://www.w3.org/WAI/ARIA/apg/)
 
-    These guidelines must be followed by U.S. websites. Other organizations such as schools are starting to follow these guidelines as well.
+    Guide to accessibility semantics defined by the Accessible Rich Internet Application (<abbr>ARIA</abbr>) specification to create accessible web experiences. Describes how to apply accessibility semantics to common design patterns and widgets, providing design patterns and functional examples.
 
-2. [<dfn>Web Content Accessibility Guidelines (WCAG)</dfn> 1.0](https://www.w3.org/TR/WAI-WEBCONTENT/)
+2. [<dfn>Web Content Accessibility Guidelines (<abbr>WCAG</abbr>)</dfn>](https://www.w3.org/WAI/standards-guidelines/wcag/)
```
Comment (teoli2003) [inline files/en-us/web/accessibility/information_for_web_authors/index.md:14]: To unify with the rest of the page:
```suggestion
2. [Web Content Accessibility Guidelines (<abbr>WCAG</abbr>)](https://www.w3.org/WAI/standards-guidelines/wcag/)
```

File: files/en-us/web/accessibility/information_for_web_authors/index.md [hunk 2]
```diff
@@ -7,63 +7,48 @@ tags:
...
+* [Accessibility Insights](https://accessibilityinsights.io/)
+* [<abbr>WAVE</abbr>](http://wave.webaim.org/extension/)
 
-    by Juicy Studio
-    How can a web author establish easily and quickly if the background color and foreground (text) color in use in his webpages have sufficient color contrast difference and sufficient color brightness difference? [Luminosity Color Contrast Ratio Analyser](https://juicystudio.com/services/luminositycontrastratio.php) is an excellent tool for such task. Effective color contrast and effective color brightness difference have a **decisive importance for reading**, furthermore for people with partial color deficiency (see the excellent examples in [Effective Color Contrast](https://lighthouseguild.org/effective-color-contrast-and-making-text-legible/) by Lighthouse International).
 
-    The [Juicy Studio CSS analyzer](https://juicystudio.com/services/csstest.php) will also analyze and report the color contrast and color brightness differences involved in your stylesheets.
+Tools to integrate into your build process , programmatically adding accessibility tests so you can catch errors as you develop your web application:
```
Comment (teoli2003) [inline files/en-us/web/accessibility/information_for_web_authors/index.md:43]: Typo
```suggestion
Tools to integrate into your build process, programmatically adding accessibility tests so you can catch errors as you develop your web application: 
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two minor typos

**Review by teoli2003 — APPROVED** [review]

**Comment by scottaohara** [comment]
@estelle since this was merged before i got a chance to look at it, if I find anything to update/add/whatever, should I instead file new issues or just make those changes myself?

**Pull request merged**